jetstream/statistics.py

```diff
@@ -166,9 +166,16 @@ def apply(
                     ref_branch_list = [reference_branch]
 
                 for ref_branch in ref_branch_list:
-                    statistic_result_collection.data += self.transform(
-                        df, metric, ref_branch, experiment
-                    ).data
+                    try:
+                        statistic_result_collection.data += self.transform(
+                            df, metric, ref_branch, experiment
+                        ).data
+                    except Exception as e:
+                        logger.error(
+                            f"Error while computing statistic {self.name} for metric {metric}: {e}",
+                            extra={"experiment": experiment.normandy_slug},
```

Comment:
This should catch exceptions for all statistics, except counts since `Count` does overwrite `apply`. But logging could also be added there if that seems useful to have

Reply:
If count fails we've got bigger problems :)